doc/onboarding/model/step.md

```diff
@@ -0,0 +1,22 @@
+# `step` State
```

Comment:
@CPatchane  This morning I changed my mind about this one. Because what we need in the view is : current step index and total steps. 
Current step index is related to the current step.
Total steps is related to the onboarding and will not change during user's course.